player.py

```diff
@@ -11,7 +11,7 @@ def get_choice(self,board):
         '''
         gets the column number from the player by user input
         '''
-        choice = int(input(f'now {self.name} pick a column-- '))
+        choice = int(input(f'now {self.name} pick a column-- ')) # input validation
```

Comment:
You're going to have a problem if the user types in anything besides a number.  A try/except block wrapped in a while True loop will resolve this.